README.md

```diff
@@ -2,7 +2,7 @@
 
 The `TagShelf.Alfred.ApiWrapper` is a comprehensive .NET library designed to facilitate seamless interactions with the Alfred API. It's tailored to support a wide range of .NET applications, from modern .NET Core and .NET Standard projects to legacy .NET Framework 4.7.2 applications, providing a robust, strongly-typed interface for efficient development.
```

Comment:
"providing a robust, strongly-typed interface for efficient development."

This phrase repeats in the next sentence.

Reply:
Rephrased